history-deleter/history.html

```diff
@@ -7,9 +7,11 @@
   </head>
 
   <body>
-    <p>History for this domain (limited to 5 results):</p>
+    <p id="history-title">History for <span id="domain1"></span> (Last 5 results) :</p>
+    <div></div>
     <p id="history"></p>
-    <p><a href="#" id="clear">Clear history for this domain</a></p>
+    <div></div>
+    <p><a href="#" id="clear">Clear history for <span id="domain2"></span></a></p>
```

Comment:
Sure, if you do a commit on the same branch it will just update this pull request.